src/components/app/Nav.svelte

```diff
@@ -18,6 +18,13 @@
       await invalidateAll();
     }
   }
+
+  function getRoleDisplayValue(userRole: UserRole) {
+    if (isAdminRole(userRole)) {
+      return 'admin';
```

Comment:
So now there are two "admin" roles in Hasura. The one provided by Hasura called "admin", and the one provided by us called "aerie_admin". I think we should probably call it "aerie_admin" so we don't confuse the two.

Reply:
But does the user need to see "aerie_admin"? Or do they just really need to know if they're "admin"

Reply:
It's different roles, so I think "aerie_admin" is more correct. Also we are going to rename user and viewer to aeire_user and aerie_viewer: https://github.com/NASA-AMMOS/aerie/issues/1021
So I think it makes sense to show the actual role names.